com.github.ytdl_gui.youtubedl_gui.yaml

```diff
@@ -0,0 +1,68 @@
+app-id: com.github.ytdl_gui.youtubedl_gui
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: youtubedl-gui
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --filesystem=host
```

Comment:
this permission is too broad without further justification. Can the app just run using the portal based access and not need any permission at all? Or be scoped to xdg-download by default?

Reply:
the app opens to xdg-download by default, it should be possible to reduce the permission to just that

Reply:
I spoke to the developer we can use portal based access